src/shared/components/person/profile.tsx

```diff
@@ -752,6 +762,42 @@ export class Profile extends Component<ProfileRouteProps, ProfileState> {
                       {capitalizeFirstLetter(I18NextService.i18n.t("unban"))}
                     </button>
                   ))}
+                {amAdmin() && (
+                  <>
+                    <button
+                      className={
+                        "d-flex registration-self-start btn btn-secondary me-2"
+                      }
+                      aria-label="View registration"
+                      onClick={this.handleRegistrationShow}
+                    >
+                      View registration
+                    </button>
+                    {showRegistrationDialog && (
+                      <DisplayModal
+                        onClose={this.handleRegistrationClose}
+                        loadingMessage="Loading registration"
+                        title={`${pv.person.display_name ?? pv.person.name}'s registration`}
```

Comment:
Since there's no good way to do this in i18n for any random string, probably just an interpolated thing like this:

`.i18n("registration_for_user", {user: pv.person.display_name ?? pv.person.name)}`

And it looks like `Registration for Bruh`